src/DotNetLightning.Core/Serialization/Features.fs

```diff
@@ -178,13 +178,13 @@ module internal Feature =
         
         
 [<StructuredFormatDisplay("{PrettyPrint}")>]
-type FeatureBits private (bitArray) =
-    member val BitArray: BitArray = bitArray with get, set
+type FeatureBits private (bitArray: BitArray) =
+    member __.BitArray
+        with get() =
```

Comment:
@aarani please note, I was actually wrong when I told you in telegram that `val` implies mutable state by default; but anyway, let's leave this like it is now, as it makes it consistent with the ByteArray property